api/envoy/api/v2/route/route.proto

```diff
@@ -903,6 +903,19 @@ message RetryPolicy {
     google.protobuf.Duration max_interval = 2 [(validate.rules).duration = {gt {}}];
   }
 
+  message RetryBudget {
+    // Specifies the limit on concurrent retries as a percentage of the sum of active requests and
+    // active pending requests.
+    //
+    // This parameter is optional. Defaults to 20%.
+    type.Percent percent_budget = 1;
+
+    // Specifies the minimum retry concurrency allowed for the retry budget.
+    //
+    // This parameter is optional. Defaults to 15.
```

Comment:
Sorry I'm still confused. Will the final result be a default of 15 * 0.2 = 3? Or will it allow 15? Can you make this doc a bit more clear?